pkg/interface/src/views/apps/chat/components/ChatWindow.tsx

```diff
@@ -136,6 +135,10 @@ export default class ChatWindow extends Component<ChatWindowProps, ChatWindowSta
       this.setState({ fetchPending: false });
     }
 
+    if (this.state.initialized !== prevState.initialized) {
+      this.dismissIfLineVisible();
+    }
```

Comment:
This is the relevant bit

Reply:
Won't this fire if state.initialized is `false` and prevState's is `true`?

Reply:
I don't think that would ever happen, but if there's some weirdness with switching chat windows, I changed it to make sure